src/main/java/services/RestoreFactory.java

```diff
@@ -351,6 +353,14 @@ private String getSyncToken(String username) {
         return storage.getMetaDataFieldForRecord(users.firstElement(), User.META_SYNC_TOKEN);
     }
 
+    // Device ID for tracking usage in the same way Android uses IMEI
+    private String getSyncDeviceId() {
+        if (asUsername == null) {
+            return DEVICE_ID_SLUG;
+        }
+        return String.format("%s|%s|as|%s", DEVICE_ID_SLUG, username, asUsername);
```

Comment:
Confirming that this won't block the current app, so let's skip for now but make sure we know about the limitation moving forward.